src/DataFramesMeta.jl

```diff
@@ -182,19 +194,20 @@ where(d::AbstractDataFrame, arg) = d[arg, :]
 where(d::AbstractDataFrame, f::Function) = d[f(d), :]
 where(g::GroupedDataFrame, f::Function) = g[Bool[f(x) for x in g]]
 
-collect_ands(x::Expr) = x
-collect_ands(x::Expr, y::Expr) = :($x & $y)
-collect_ands(x::Expr, y...) = :($x & $(collect_ands(y...)))
+function and(x, y)
+    if VERSION < v"0.6.0-"
+        :($x & $y)
+    else
+        :($x .& $y)
+    end
+end
```

Comment:
It would be better to move the conditional outside of the function definition, i.e.
```julia
if VERSION < v"0.6.0-"
    and(x, y) = :($x & $y)
else
    and(x, y) = :($x .& $y)
end
```
Otherwise the version check is performed every time the function is called.